docs/tutorials/code/craft-a-snap/snapcraft-01-basic.yaml

```diff
@@ -0,0 +1,29 @@
+name: ukuzama-pyfiglet
+base: core24
+version: "0.1"
+summary: pyfiglet is a program for making large letters out of ordinary text
+description: |
+  pyfiglet is a full port of FIGlet (http://www.figlet.org/) into pure python.
+  It takes ASCII text and renders it in ASCII art fonts (like the title above,
+  which is the 'block' font).
+
+  This snap is not endorsed by the pyfiglet project.
+
+platforms:
+  amd64:
+
+grade: devel
+confinement: devmode
+
+parts:
+  pyfiglet:
+    plugin: python
+    source-type: git
+    source: https://github.com/medubelko/pyfiglet
+    stage-packages: [python3-minimal, python3.12-minimal]
+
+apps:
+  ukuzama-pyfiglet:
+    command: bin/pyfiglet
+    environment:
+      PYTHONPATH: /snap/ukuzama-pyfiglet/x1/lib/python3.12/site-packages
```

Comment:
When installed in the spread container, the Python paths weren't picking up the local snap's site-packages. This is a workaround for it, but I don't like how fragile it is.